src/entities/purchase-params.ts

```diff
@@ -21,4 +21,15 @@ export interface PurchaseParams {
    * The email of the user. If undefined, RevenueCat will ask the customer for their email.
    */
   customerEmail?: string;
+
+  /**
+   * The locale to use for the purchase flow. If not specified, the device's locale will be used.
+   */
+  selectedLocale?: string;
```

Comment:
It might be worth to write an E2E test verifying that localization is working, even if it's a basic one.

Reply:
I just added one!

Both with and without paywall!